assets/src/pages/room/RoomPage.tsx

```diff
@@ -98,17 +100,34 @@ const RoomPage: FC<Props> = ({
     messageComparator
   );
 
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
   return (
     <PageLayout>
       <div className="flex h-full w-full flex-col gap-y-4">
         {/* main grid - videos + future chat */}
-        <section className="flex h-full w-full flex-col">
+        <section
+          className={clsx(
+            "flex h-full w-full self-center justify-self-center 3xl:max-w-[1728px]",
+            isSidebarOpen && "gap-x-4"
+          )}
+        >
           <VideochatSection
             peers={peerState.remote}
             localPeer={peerState.local}
             showSimulcast={showSimulcastMenu}
             webrtc={webrtc}
           />
+
+          <div
+            className={clsx(
+              isSidebarOpen ? "max-w-[300px]" : "max-w-0",
```

Comment:
There is a bug when you try to narrow the page with the sidebar open. 

https://user-images.githubusercontent.com/58188916/220872469-07776ac5-35a8-4d03-94f5-fe881e7f0206.mov


There also need to be another condition about the width this section brings or an auto sidebar closing.